lib/install/validate/config_to_data.go

```diff
@@ -72,13 +72,19 @@ func SetDataFromVM(ctx context.Context, finder Finder, vm *vm.VirtualMachine, d
 	if err != nil {
 		return err
 	}
-	rp, ok := or.(*object.ResourcePool)
-	if !ok {
+
+	// we are attempting to present the resource pool inventory path
+	// in a DRS disabled environment that inventory path could point to a
+	// cluster, so we need to evaluate the type and set the path accordingly
+	switch r := or.(type) {
```

Comment:
Can this not be cast to `object.Common` as that contains the inventory path and is embedded in both types.

Reply:
Yes -- there's a better way to do it